src/components/PodcastCard/PodcastCard.js

```diff
@@ -24,23 +51,65 @@ const Card = styled.section`
   display: flex;
   flex-direction: column;
   gap: 1rem;
+  cursor: pointer;
   border-radius: 16px;
   padding: 0.75rem 1rem;
-  width: min(45%, 40rem);
+  width: ${props => (props.clicked ? '90%' : 'min(60%, 40rem)')};
   box-shadow: rgba(0, 0, 0, 0.25) 0px 14px 15px,
     rgba(0, 0, 0, 0.12) 0px -12px 30px, rgba(0, 0, 0, 0.12) 0px 4px 6px,
     rgba(0, 0, 0, 0.17) 0px 12px 13px, rgba(0, 0, 0, 0.09) 0px -3px 5px;
 `
+
+const CardMainContainer = styled.div`
+  display: flex;
+  flex-direction: ${props => (props.clicked ? 'row' : 'column')};
+  gap: 1rem;
+`
+
+const InnerContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+`
+
 const Image = styled.img`
   align-self: center;
 `
+const Author = styled.h3`
+  display: ${props => (props.clicked ? '' : 'none')};
+  font-size: clamp(0.5rem, -0.9rem + 8.333vw, 3.5rem);
+  text-align: center;
+`
 
 const Title = styled.h2`
   text-align: center;
-  font-size: clamp(0.5rem, -0.875rem + 8.333vw, 3.5rem);
+  font-size: clamp(0.5rem, -0.7rem + 8.333vw, 3rem);
```

Comment:
Just a testing suggestion: Try changing your browsers font size (not sure wether just zooming in would do the trick) since rem is affected by the base font-size. 

`calc(-0.875rem + 8.333vw)` is probably needed here.

Reply:
great input i will test that